src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+header {
+  padding: 0 50px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: flex-end;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin: auto;
+}
+
+.links {
+  height: 15px;
+  display: flex;
+  list-style-type: none;
+  justify-content: flex-end;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+  margin: auto;
```

Comment:
again, why do you need margin: auto?

Reply:
I changed on margin 0, but without margin entry it doesn`t work (